service-loadbalancer/README.md

```diff
@@ -15,6 +15,35 @@ This module will not help with the latter
 
 ## Overview
 
+The image below roughly describes the service-loadbalancer alternative (compared with using NodePorts).
+
+```
+                          +----------------------+                     
+                          | haproxy load balancer|                     
```

Comment:
this box is actually service loadbalancer, not haproxy. labeling it service loadbalancer implies some lb is being used, but the reverse is not true.